ThermoFun/Database.cpp

```diff
@@ -426,32 +426,32 @@ auto Database::mapReactions() const -> const ReactionsMap&
     return pimpl->mapReactions();
 }
 
-auto Database::getElements() -> std::vector<Element>
+auto Database::getElements() const -> std::vector<Element>
 {
     return pimpl->getElements();
 }
 
-auto Database::getSubstances() -> std::vector<Substance>
+auto Database::getSubstances() const -> std::vector<Substance>
```

Comment:
We can't return a const reference to this vector because it gets built on the fly.